orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/providers/kubernetes/KubernetesContainerFinder.groovy

```diff
@@ -20,6 +20,33 @@ import com.netflix.spinnaker.orca.pipeline.model.Pipeline
 import com.netflix.spinnaker.orca.pipeline.model.Stage
 
 class KubernetesContainerFinder {
+  static Map parseContainerPartsFrom(String containerName) {
+    String tag = ""
+    String registry = ""
+
+    String[] parts = containerName.split('/')
+
+    String head = parts.first()
+
+    // Get the registry
+    if (head.contains('.') || head.startsWith('localhost')) {
```

Comment:
Good question. My thoughts were that if no registry is defined the k8s kluster should chose a default. 

In retrospect however I'm entirely sure. I think that assuming some default here is hard, so we would either have to use a registry defined for this account or fail if we don't allow this to be empty. Wdyt?